tile-map/main.py

```
@@ -0,0 +1,140 @@
+			with open(file_path, "wb") as file:
+				file.write(img_binary.tobytes())
+			return True
+
+		elif response.status_code == 429:
+			# 429 means "Too Many Requests". Sleep this thread to cool off.
+			time.sleep(5.0)
+			return False
+
+		else:
+			return False
+	except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)